firestore/cloud-client/query_filter_or_test.py

```diff
@@ -55,15 +59,40 @@ def delete_document_collection(data, collection):
 
 
 @backoff.on_exception(backoff.expo, Exception, max_tries=3)
-def test_query_or_composite_filter(capsys, data):
+def test_query_or_filter(capsys, data):
     client = firestore.Client(project=PROJECT_ID)
-    collection = client.collection("users")
+    collection = client.collection("cities")
 
     try:
         create_document_collection(data, collection)
-        query_or_composite_filter(PROJECT_ID)
+        query_or_filter(PROJECT_ID)
     finally:
         delete_document_collection(data, collection)
 
     out, _ = capsys.readouterr()
```

Comment:
Testing on stdout is brittle and can cause tests to fail if the underlying implementation changes. Instead, you should return the response from the code snippet and test it directly here.

Reply:
I'm hoping to avoid re-writing the existing samples and tests for this PR